src/translations/i18n.js

```diff
@@ -1,31 +1,37 @@
 import { createI18n } from 'vue-i18n';
-import { enTranslations, enUSTranslations, esTranslations, esCOTranslations } from './exports';
+import { enTranslations, enUSTranslations, esTranslations, esCOTranslations, deTranslations } from './exports';
 
 export const languageOptions = {
   'en-US': { translations: enUSTranslations, language: 'English (United States)', code: 'usa' },
   en: { translations: enTranslations, language: 'English (United Kingdom)', code: 'gb' },
   es: { translations: esTranslations, language: 'Español (Spain)', code: 'es' },
   'es-CO': { translations: esCOTranslations, language: 'Español (América Latina)', code: 'col' },
+  de: { translations: deTranslations, language: 'Deutsch', code: 'de' },
 };
-export let browserLocale = window.navigator.language;
 
-const getLocale = (locale) => {
-  if (Object.keys(languageOptions).includes(locale)) {
-    console.log('Locale found in languageOptions: ', locale);
-    return locale;
-  } else if (locale.includes('es')) {
-    console.log('Spanish dialect not supported, using default es.');
-    return 'es';
+const browserLocale = window.navigator.language;
+const isLevante = import.meta.env.MODE === 'LEVANTE';
+
+const getLocale = (localeFromBrowser) => {
+  const localeFromStorage = sessionStorage.getItem(`${isLevante ? 'levante' : 'roar'}PlatformLocale`);
+
+  if (localeFromStorage) {
+    return localeFromStorage;
   } else {
-    console.log('Language not supported, using default en-US.');
-    return 'en-US';
+    sessionStorage.setItem(`${isLevante ? 'levante' : 'roar'}PlatformLocale`, localeFromBrowser);
```

Comment:
Same question as above: why does this need to be differentiated between ROAR and LEVANTE?